src/main/java/mate/controller/LoginController.java

```diff
@@ -0,0 +1,43 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import mate.exception.AuthenticationException;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.AuthenticationService;
+
+public class LoginController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private static final String DRIVER_ID = "driver_id";
+    private static final String NAME = "name";
+    private final AuthenticationService authenticationService
+            = (AuthenticationService) injector.getInstance(AuthenticationService.class);

```

Comment:
If you want you can try this style of code. As for me is a little bit readable (like streams).
`private final AuthenticationService authenticationService = (AuthenticationService) injector
.getInstance(AuthenticationService.class);`